src/raised-button.jsx

```diff
@@ -408,7 +408,7 @@ const RaisedButton = React.createClass({
 
     return (
       <Paper
-        className={this.props.className}
```

Comment:
Did you mean to include these `className` changes in this PR? Best to keep unrelated changes separate as it gives the option to revert one, leaving the other.